include/sound/sof.h

```diff
@@ -97,8 +97,5 @@ struct sof_dev_desc {
 };
```

Comment:
@ranj063 Again a nit with the commit message: "Set the drv_name and tplg_filename.... in the sof_machine_check op" However this patch does not touch sof_machine_check, making this confusing.

Did you mean that "As the drv_name and tplg_filename for nocodec machine driver is set in the sof_machine_check, sof_nocodec_setup() does not need..." ?

Reply:
@kv2019i I changed the commit message slightly but the patch does modify sof_machine_check() to set the drv_name for the nocodec case no?

Reply:
@ranj063 Ah, you are right, I misread. Having rationale why the change is done would be good as well. So I guess here the idea is to decouple sof_nocodec_setup() from acpi mach structure, right? But yeah, this is apparent from the series, so good enough for me.